tests/control/ros/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Add a check here to only build these tests if the `ros::control` component was built.